packages/vsce/src/utils/plexUtils.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+export interface ICicsPlexInfo {
```

Comment:
Just for future reference, this is temporary and will be replaced by an SDK interface.